### PR TITLE
Feature/revise pkg download

### PR DESCRIPTION
**Pull request by jhoobergs**

### CODE REVIEW

**Comment by jhoobergs** [comment]
Fixes #358

File: api/services/DownloadStatsService.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 var _ = require('lodash');
 var Promise = require('bluebird');
 var dateFormat = require('dateformat');
+var CSV = require('csv-js');
```
Comment (ludov04) [inline api/services/DownloadStatsService.js]: you need to also add thoses package to package.json

Comment (jhoobergs) [inline api/services/DownloadStatsService.js]: Probably removed it when removing other packages I added but didn't need eventually

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "cheerio": "^0.20.0",
     "connect-flash": "^0.1.1",
     "connect-redis": "^3.1.0",
+    "csv-js": "^1.0.0",
```
Comment (jhoobergs) [inline package.json]: @ludov04  Seems like I already did that

Comment (ludov04) [inline package.json]: I missed it somehow, but `zlib` is still missing

Comment (jhoobergs) [inline package.json]: @ludov04  `zlib` is part of node

**Pull request merged**